frontend/src/components/Modals/DetailsButtonModal.tsx

```diff
@@ -47,6 +49,9 @@ export const DetailsButtonModal = ({
     <Modal show={show} onHide={handleClose} fullscreen={true}>
       <Modal.Header closeButton>
         <Modal.Title>
+          <small className="text-muted ps-2">
+            JSON Schema test cases that need attention : &nbsp;
```

Comment:
```suggestion
            Unsuccessful Tests:
```

Reply:
The changed code needs to be &nbsp ; follow Unsuccessful Tests: 

Before the inclusion of &nbsp ;

<img width="498" alt="Screenshot 2024-02-21 at 10 44 01 AM" src="https://github.com/bowtie-json-schema/bowtie/assets/112953828/c4e4513a-48f8-459b-9da7-86ff6b0951d2">

After  the inclusion of &nbsp ;

<img width="479" alt="Screenshot 2024-02-21 at 10 44 33 AM" src="https://github.com/bowtie-json-schema/bowtie/assets/112953828/759a6cef-9a77-47b3-8776-e05c838b839b">


Note the space between : and Jsonschema

Reply:
I certainly can use labels with padding properties. The project has not used label tag on any screen so far and I tried to keep my fixes in synch with the existing coding style. I will in the future with your permission do what is right for my fixes even if it looks slightly out of place. Sorry about the back and forth.